apps/sms/views/conversation/js/conversation.js

```diff
@@ -80,6 +80,11 @@ var ConversationView = {
   showErrorInFailedEvent: '',
   previousSegment: 0,
 
+  /**
+   * Set to true after init is run.
+   */
+  _isReady: false,
```

Comment:
it could work here but it works only for synchronous operations.

I thought that Navigation could lazy load the views (or more appropriately have View mocks until they loaded, whose beforeEnter method would load them), then load init, and only then resolve the promise when init returns or the returned promise is resolved (or something like this).